### ISSUE TITLE
Use relative times in the UI

### ISSUE THREAD

**Issue by w0rm** [prometheus/alertmanager#810]
When alerts are hitting, we're probably interested in how many minutes/seconds ago it started, and not a UTC timestamp. The same goes for silences. E.g. for the active silences we're interested in for how long they remain active.

So far I could only find this library http://package.elm-lang.org/packages/alpacaaa/elm-date-distance/latest that implements this formatting. We also need to get rid of `jweir/elm-iso8601` in favor of `rluiten/elm-date-extra` to keep our deps at a minimum.

**Comment by mxinden** [comment]
I want to add this comment by Jack (See [google group](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/prometheus-users/QBTPLmB5c2o/RGDKytieAwAJ)) here:

> Time stamps are in UTC.  I prefer this frankly, but wish they where labeled as such.  (I work across many timezones and there is only one true time.)  But I imagine others have different opinions.

I am fine with either one. Any other opinions @stuartnelson3 @fabxc @brancz?

**Comment by brancz** [comment]
I think I'd like the relative time in the UI, but the real UTC timestamp when I hover over it (as @jjneely mentioned it would be good to make clear that it is indeed UTC).

@jjneely what do you think about this?

**Comment by w0rm** [comment]
Yeah, we can show UTC timestamp in a tooltip https://v4-alpha.getbootstrap.com/components/tooltips/#static-demo

**Comment by stuartnelson3** [comment]
I would prefer the duration. Assuming the duration calculation is correct, it saves me the trouble of doing the mental math to move from the reported UTC time to my current time.

**Comment by brancz** [comment]
@stuartnelson3 I believe you are talking about silences rather than alerts?

**Comment by stuartnelson3** [comment]
Yeah, silences

**Comment by m-masataka** [comment]
Ping!
Was this issue conclusion reached?
I would prefer the duration, to avoid mental calculation that move timestamp from UTC to local timezone.

As other solution, I want to select the timezone (only in UI).
